src/hooks/xcm/useXcmBridge.ts

```diff
@@ -103,11 +102,15 @@ export function useXcmBridge(selectedToken: Ref<Asset>) {
   );
 
   const isMoonbeamWithdrawal = computed<boolean>(() => {
-    return destChain.value.name === Chain.MOONRIVER || destChain.value.name === Chain.MOONBEAM;
+    // Todo: un-comment-out after channel between Astar and Moonbeam has been opened
+    // return destChain.value.name === Chain.MOONRIVER || destChain.value.name === Chain.MOONBEAM;
+    return destChain.value.name === Chain.MOONRIVER;
   });
 
   const isMoonbeamDeposit = computed<boolean>(() => {
-    return srcChain.value.name === Chain.MOONRIVER || srcChain.value.name === Chain.MOONBEAM;
+    // Todo: un-comment-out after channel between Astar and Moonbeam has been opened
+    // return srcChain.value.name === Chain.MOONRIVER || srcChain.value.name === Chain.MOONBEAM;
+    return srcChain.value.name === Chain.MOONRIVER;
```

Comment:
I noticed that adding direct references to this script is not a scalable solution. We should keep the code composable by isolating chain config to different chains

Reply:
Noted, currently, I'm working on the new transfer page and this hook is refactored too. How about let me modify in the branch of the new transfer page? Tbh, this is a bit challenging but I'll try to clean up more.

Reply:
@hoonsubin Please help take a look at my comment when you are available 🙏

Reply:
Then what about adding this to our backlog for later?

Reply:
ok!